must-gather/collection-scripts/gather_ceph_resources

```diff
@@ -191,20 +191,36 @@ for ns in $namespaces; do
         done
     done
 
-    # Add Ready nodes to the list
+    #Add ocs ready nodes to the list
     nodes=$(oc get nodes -l cluster.ocs.openshift.io/openshift-storage='' --no-headers | awk '/\yReady\y/{print $1}')
 
-    # Collecting ceph prepare volume logs
+    for i in $(timeout 120 oc -n "${ns}" exec "${HOSTNAME}"-helper -- bash -c "ceph crash ls --connect-timeout=15"| awk '{print $1}'); do
+        { timeout 120 oc -n "${ns}" exec "${HOSTNAME}"-helper -- bash -c "ceph crash info $i --connect-timeout=15" >> "${COMMAND_OUTPUT_DIR}"/crash_"${i}"; } >> "${BASE_COLLECTION_PATH}"/gather-debug.log 2>&1;
+    done
+
+    oc delete -f pod_helper.yaml
+
     volume_collection(){
+        NODE_OUTPUT_DIR=${CEPH_COLLECTION_PATH}/volume_collection_${node}
+        mkdir -p "${NODE_OUTPUT_DIR}"
         printf "collecting prepare volume logs from node %s \n"  "${node}"
         oc rsync -n "${ns}" "$(oc get pods -n "${ns}"| grep "${node//./}-debug"| awk '{print $1}')":/host/var/lib/rook/openshift-storage/log "${NODE_OUTPUT_DIR}"
     }
 
-    for i in $(timeout 120 oc -n "${ns}" exec "${HOSTNAME}"-helper -- bash -c "ceph crash ls --connect-timeout=15"| awk '{print $1}'); do
-        { timeout 120 oc -n "${ns}" exec "${HOSTNAME}"-helper -- bash -c "ceph crash info $i --connect-timeout=15" >> "${COMMAND_OUTPUT_DIR}"/crash_"${i}"; } >> "${BASE_COLLECTION_PATH}"/gather-debug.log 2>&1;
-    done
+    journal_collection(){
+        JOURNAL_LOGS=${CEPH_COLLECTION_PATH}/journal_log_${node}
+        mkdir -p "${JOURNAL_LOGS}"
+        printf "collecting journal logs from %s \n" "${node}"
+        oc rsync -n "${ns}" "$(oc get pods -n "${ns}"| grep "${node//./}-debug"| awk '{print $1}')":/host/var/journal.log "${JOURNAL_LOGS}"
+    }
 
-    oc delete -f pod_helper.yaml
+    kernel_collection(){
+        KERNEL_LOGS=${CEPH_COLLECTION_PATH}/kernel_log_${node}
+        mkdir -p "${KERNEL_LOGS}"
+        printf "collecting kernel logs from %s \n" "${node}"
+        oc rsync -n "${ns}" "$(oc get pods -n "${ns}"| grep "${node//./}-debug"| awk '{print $1}')":/host/var/log/messages "${KERNEL_LOGS}"
+        oc rsync -n "${ns}" "$(oc get pods -n "${ns}"| grep "${node//./}-debug"| awk '{print $1}')":/host/var/log/sysstat "${KERNEL_LOGS}"
+    }
```

Comment:
move this also to clustered_scope file

Reply:
No, we cannot do it. Because we are copying files where we are putting debug pods into the background. In cluster_scope file, WE ARE NOT putting debug pods into the background, we are just creating journal log inside the node and exiting right away.

Reply:
So the thing is we want node data to be specific dir. So as it easy to locate.

Reply:
There are pros and cons to organizing things in directories per-node versus having all the logs in one place. I think for the purposes of debugging Ceph it makes most sense to have them all in one place, but I could be convinced otherwise.

Reply:
My take on this would be lets keep it this way and the this in. And in the next iteration, we can refactor out the necessary changes. We are also planning to stat debug pods as soon as the must gather starts so in that way, we do not have to worry about putting all the debug pods related operations (`oc rsync` in this case) at one place?.
This code works btw, we were only waiting got your decision @jarrpa